### PR TITLE
Landing page update with scope.md references

### PR DESCRIPTION
**Pull request by rtorrero**
This is a re-submission of https://github.com/trento-project/trento/pull/78

(see https://github.com/trento-project/trento/pull/78#issuecomment-859355825)

### CODE REVIEW

File: web/layout.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ type SubmenuItem struct {
 }
 
 var defaultLayoutData = LayoutData{
-	Title:     "Trento Console",
+	Title:     "Trento Console for SAP Applications",
```
Comment (dirkmueller) [inline web/layout.go:40]: why this change? it is already long enough imho ;)

File: web/templates/home.html.tmpl [hunk 2]
```diff
@@ -4,4 +4,113 @@
...
+<div class='container'>
+    <div class='card-deck'>
+        <div class='card'>
+            <h5 class='card-header'>Agents</h5>
+            <div class='card-body'>
+                <p class='card-text'>
+                    Though not specifically a view/section, as it is the core component of Trento,
+                    it is important to understand what they are.
+
+                    The agents are the processes that run in each of the nodes that conform
```
Comment (dirkmueller) [inline web/templates/home.html.tmpl:16]: this (and the others) duplication of text with the docs/ file is a bit unfortunate. how difficult would it be to pull something like https://github.com/gomarkdown/markdown to embed the documentation ?

Comment (stefanotorresi) [reply web/templates/home.html.tmpl:16]: We thought about this, and we already have `gomarkdown/markdown` integrated and available via the `markdown` template function (see #62).
That said, the homepage is likely to change very soon anyway, so we agreed that it would make a simple static HTML page more complicated than it really should.

File: web/layout.go (hunk 1 above)
Comment (rtorrero) [reply web/layout.go:40]: I thought I saw it in some of the UI mocks but I can't seem to find it anymore. I'll change it!

File: web/templates/home.html.tmpl (hunk 2 above)
Comment (rtorrero) [reply web/templates/home.html.tmpl:16]: @dirkmueller I already explored this and had a rudimentary version working. But if we want to keep markdown for the scope document it gets quite ugly as one needs to iterate over markdown (or what I did, convert to html and navigate the <h3> tags to render each one of them in a separate card). `JSON` or `yaml` would be better for this but at the expense of losing readability in the document when accessed separately; but IMO this would be overcomplicating this for something that is just meant to be temporal anyways.

**Review by stefanotorresi — CHANGES_REQUESTED** [review]
Hey team, tlease see https://github.com/trento-project/trento/discussions/86 before merging.

Beside that, I think the sections should be ordered according to the sidebar (also see #84 about that), which would put the "Agents" section last, which also reflects the source documentation structure.

Also, I'm having second thoughts about the Call To Action link going to the docs: I would expect the main action of the user, once they open the homepage, to be going to any of the main navigation items of the application. 

We might be misinterpreting Sorin's original indications: the Call To Action of each section should go to the relevant item, while we should have one single link to the source documentation at the bottom of the page. This also means that we should produce a meaningful abstract text instead of letting Bootstrap's automatic ellipsis truncate the text with `...`.

File: web/templates/home.html.tmpl [hunk 3]
```diff
@@ -4,4 +4,113 @@
     <h1>{{ .Title }}</h1>
     <p>{{ .Paragraph }}</p>
 </div>
+<div class='container'>
```
Comment (en3sis) [inline web/templates/home.html.tmpl:7]: The `container` class will add some max-width and extra margins and it only should be used once.

We usually divide the content of a section in `<section>  </section>` that has no extra style and it won't push the content related to the other elements (titles or descriptions)

**Comment by en3sis** [comment]
@stefanotorresi the way it was implemented would be the correct one based on the very early discussion that was about having a quick description of each functionality where we would talk about the current status (brief explanation) and a link to the documentation for more details.
If the scope changed we could rework it, definitely. 

> This also means that we should produce a meaningful abstract text instead of letting Bootstrap's automatic ellipsis truncate the text with   

Agree :P 

> single link to the source documentation at the bottom of the page

Would be also useful

**Comment by rtorrero** [comment]
I've done the requested changes, some of them have a little interpretation on my side; let me know if something else needs change.
@en3sis about using `<section>`, the thing is that the margins don't exactly match what you did in your mock. I think it's close enough but be sure to check it. Also, the fonts are not exactly the same, let me know what bootstrap classes I need to use if you'd like to have the same. Thanks!
landing2

**Comment by en3sis** [comment]
@rtorrero I think the only thing it's missing is that the  `.subheadline` class for the subheadline.

For the fonts please refer to https://productbrand.suse.com/typography but basically, all headers should use `Poppins` while `p` tags should use `Lato`. (it should be part of the DS CSS already so no extra work should be needed) 

Everything else LGTM!

**Comment by rtorrero** [comment]
@en3sis Thanks! I guess the differences I saw on my local tests vs your screenshot mockup are due to OS?

@stefanotorresi should I squash or do you see anything else that needs change? I used the mockup text as I find them a good starting point.

File: web/home.go [hunk 4]
```diff
@@ -13,8 +13,9 @@ type HomeData struct {
 
 func HomeHandler(c *gin.Context) {
 	data := HomeData{
-		Title:     defaultLayoutData.Title,
-		Paragraph: "This is the home page",
+		Title: defaultLayoutData.Title,
+		Paragraph: `An open cloud-native web console improving on the
```
Comment (arbulu89) [inline web/home.go:17]: I find this `on` from `...console improving on the...` out of place there

File: web/home.go [hunk 5]
```diff
@@ -13,8 +13,9 @@ type HomeData struct {
 
 func HomeHandler(c *gin.Context) {
 	data := HomeData{
-		Title:     defaultLayoutData.Title,
-		Paragraph: "This is the home page",
+		Title: defaultLayoutData.Title,
```
Comment (arbulu89) [inline web/home.go:16]: As commented, I think the `Title` and `Paragraph` should go in the html template itself, together with everything else, but well, who cares, we will touch this many time on the road hehe

**Review by arbulu89 — APPROVED** [review]
It looks great for me. Added some really non meaningful comments hehe

PD: I guess the `SAP systems` text in the sidebar is changed in other PR, right?

File: web/home.go (hunk 4 above)
Comment (rtorrero) [reply web/home.go:17]: I think both are right, but since your suggestion has two characters less... I agree :-p

File: web/home.go (hunk 5 above)
Comment (rtorrero) [reply web/home.go:16]: The title is also rendered in the menu so I'll leave it, but I see no point in keeping the paragraph there. I'll put it in the template. Thanks!

**Review by stefanotorresi — APPROVED** [review]

**Comment by stefanotorresi** [comment]
gj 🍺
